### PR TITLE
2017-06-19 added to Fall Summit description

### PR DESCRIPTION
**Pull request by NewMexicoKid**
Added four new subsections to the Fall Summit 2017 page (defining InnerSource, the Commons, who should attend and why).

### CODE REVIEW

File: events/isc-fall-2017.md [hunk 1]
```diff
@@ -24,6 +24,22 @@ Some examples of possible content:
...
 
+### What is InnerSource?
+InnerSource takes the lessons learned from developing open source software and applies them to the way companies develop software internally. As developers have become accustomed to working on world class open source software, there is a strong desire to bring those practices back inside the firewall and apply them to software that companies may be reluctant to release. For companies building mostly closed source software, InnerSource can be a great tool to help break down silos, encourage internal collaboration, accelerate new engineer on-boarding, and identify opportunities to contribute software back to the open source world.
+ 
+### What is the InnerSource Commons?
+ InnerSourceCommons.org (aka ISC) utilizes open source methods to provide organizations pursuing inner sourcing a forum for discussing and improving the practice of inner source through the sharing of experiences (under the [Chatham House Rule](https://www.chathamhouse.org/about/chatham-house-rule)), creation and review of inner source patterns, and the open exchange of ideas.
```
Comment (ErinMB) [inline events/isc-fall-2017.md]: Should we include: 
InnerSource Commons is a consortium of representatives from over 60 companies and organizations...

**Comment by ErinMB** [comment]
Just added 2 comments

**Comment by ErinMB** [comment]
I saw in the planning document that Vijay Gurbani has confirmed his keynote - should we add that to this page? It might help draw some people to the event. Also - will Vijay link to ISC.org from any of his comms? That could also help.

**Comment by NewMexicoKid** [comment]
Thanks, Erin. I've added your comments into the PR.

**Pull request merged**